api/buckets/buckets.go

```diff
@@ -22,20 +37,56 @@ var (
 )
 
 type Bucket struct {
-	Key       string `json:"_id"`
-	Name      string `json:"name"`
-	Path      string `json:"path"`
-	DNSRecord string `json:"dns_record,omitempty"`
-	CreatedAt int64  `json:"created_at"`
-	UpdatedAt int64  `json:"updated_at"`
+	Key       string   `json:"_id"`
+	Name      string   `json:"name"`
+	Path      string   `json:"path"`
+	DNSRecord string   `json:"dns_record,omitempty"`
+	Archives  Archives `json:"archives"`
+	CreatedAt int64    `json:"created_at"`
+	UpdatedAt int64    `json:"updated_at"`
+}
+
+type Archives struct {
+	Current Archive   `json:"current"`
+	History []Archive `json:"history"`
+}
```

Comment:
Mongo world:
We save the current archive (latest), and also the history. The history isn't used yet but sounds like it's useful.